spec/unit/lib/email_spec.rb

```diff
@@ -40,90 +43,12 @@
   end
 
   describe '.find' do
-    let(:mailgun_message){ double(to_h: message) }
-    let(:events) { double(to_h: {'items' => [event]}) }
+    let(:recipient) { 'test@user.com' }
     subject { Email.find(recipient, message_subject) }
-    context 'when message is found' do
-      let(:event) { {'message' => {'recipients' => [recipient], 'headers' => {'subject' => message_subject} }, 'storage' => {'key' => '1234567890'} } }
-      before do
-        allow(Mailgun::Connector.instance.client).to receive(:get).with('mailgun@test.domain/events', event: 'stored').ordered.once {events}
-        allow(Mailgun::Connector.instance.client).to receive(:get).with('domains/mailgun@test.domain/messages/1234567890').ordered.once { mailgun_message }
-      end
-      it do
-        expect(Email).to receive(:new).with(message).once
-        subject
-      end
-    end
-    context 'when message is not found' do
-      let(:event) { {'message' => {'recipients' => ['other@test.com'], 'headers' => {'subject' => message_subject} }, 'storage' => {'key' => '1234567890'} } }
-      before do
-        allow(settings).to receive(:timeout_small) { 0.5 }
-        allow(settings).to receive(:timeout_short) { 0.05 }
-        allow(Mailgun::Connector.instance.client).to receive(:get).with('mailgun@test.domain/events', event: 'stored').at_least(:twice).ordered {events}
-      end
-      it do
-        expect(log).to receive(:error).with(Howitzer::EmailNotFoundError, "Message with subject '#{message_subject}' for recipient '#{recipient}' was not found.")
-        subject
-      end
-    end
-  end
-
-  describe '#plain_text_body' do
-    it { expect(email_object.plain_text_body).to eql message['body-plain'] }
-  end
-
-  describe '#html_body' do
-    it { expect(email_object.html_body).to eql message['stripped-html'] }
-  end
-
-  describe '#text' do
-    it { expect(email_object.text).to eql message['stripped-text'] }
-  end
-
-  describe '#mail_from' do
-    it { expect(email_object.mail_from).to eql message['From'] }
-  end
-
-  describe '#recipients' do
-    subject { email_object.recipients }
-    it { is_expected.to be_a_kind_of Array }
-
-    context 'when one recipient' do
-      it { is_expected.to include message['To']}
-    end
-
-    context 'when more than one recipient' do
-      let(:second_recipient) { 'second_tester@gmail.com' }
-      let(:message_with_multiple_recipients) { message.merge({'To' => "#{recipient}, #{second_recipient}"}) }
-      let(:email_object) { Email.new(message_with_multiple_recipients) }
-      it { is_expected.to eql [recipient, second_recipient] }
+    it do
```

Comment:
before { xpect(Email.adapter).to receive(:find).with(recipient, message_subject).once {true}}
it { is_expected.to eq(true) }